go/sqltypes/arithmetic.go

```diff
@@ -40,6 +42,91 @@ type numeric struct {
 
 var zeroBytes = []byte("0")
 
+//Addition adds two values together
+//if v1 or v2 is null, then it returns null
+func Addition(v1, v2 Value) (Value, error) {
+	if v1.IsNull() {
+		return NULL, nil
+	}
+
+	if v2.IsNull() {
+		return NULL, nil
+	}
```

Comment:
As an sql-guy I find it easier to follow as if v1 or v2 is null return:
```
if v1.IsNull() || v2.IsNull() {
  return NULL, nil
}
```
(You don't have to follow my suggestion.. another reviewer may disagree).

Reply:
I agree with this change.